tekton/release-cheat-sheet.md

```diff
@@ -6,13 +6,20 @@ the pipelines repo, a terminal window and a text editor.
 
 1. `cd` to root of Pipelines git checkout.
 
-2. Point `kubectl` at dogfooding cluster.
+1. Configure `kubectl` to use the  dogfooding cluster.
 
     ```bash
+    # This will update your kubectl to automatically connect to the dogfood cluster.
+    # Remember to point it back to your own cluster afterward, or configure acesss
+    # to multiple clusters: https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/
```

Comment:
I don't think the steps you linked to are really an "or". The gcloud command does exactly that - it fetches the auth information, creates a new cluster entry in your kubectl, and then also swaps the current cluster over to that one. You need to do that the first time you use the dogfood cluster, but you can use normal kubectl config commands every time after that.

Reply:
kk ill try to make this more accurate